templates/csharp/api-plugin-from-scratch-bearer/Repair.cs.tpl

```diff
@@ -2,10 +2,11 @@ using Microsoft.Azure.Functions.Worker;
 using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using System.Net;
```

Comment:
It looks like change class/variable name, why does it need to include new library of System.Net?

Reply:
Why this class is Repair and that in templates/csharp/api-plugin-from-scratch-oauth/Repairs.cs.tpl is Repairs, can we keep them aligned?

Reply:
If we don't have this new library, it will cause HttpStatusCode not to be found.